<launch>

  <arg name="s_init" default="0" />
  <arg name="v_init" default="0" />
  <arg name="a_init" default="0" />
  <arg name="s_final" default="0.5" />
  <arg name="v_final" default="0.0" />
  <arg name="a_final" default="0.0" />
  <arg name="v_max" default="1" />
  <arg name="a_max" default="2" />
  <arg name="j_max" default="10" />
  

  <node name="adaptative_s_curve" pkg="spline_problems" type="easy_s_curve" output="screen" >
    <param name="s_init" value="$(arg s_init)" />
    <param name="v_init" value="$(arg v_init)" />
    <param name="a_init" value="$(arg a_init)" />
    <param name="s_final" value="$(arg s_final)" />
    <param name="v_final" value="$(arg v_final)" />
    <param name="a_final" value="$(arg a_final)" />
    <param name="v_max" value="$(arg v_max)" />
    <param name="a_max" value="$(arg a_max)" />
    <param name="j_max" value="$(arg j_max)" />
  </node>
</launch>
